onnxruntime/test/providers/qnn/max_pool_test.cpp

```diff
@@ -247,79 +159,66 @@ TEST_F(QnnCPUBackendTests, DISABLED_TestMaxPool_Large_Input2_Ceil) {
+                                utils::MakeAttribute("dilations", std::vector<int64_t>{1, 1}),
+                                utils::MakeAttribute("ceil_mode", static_cast<int64_t>(1)),
+                                utils::MakeAttribute("storage_order", static_cast<int64_t>(0)),
+                                utils::MakeAttribute("auto_pad", "NOTSET")},
                                ExpectedEPNodeAssignment::All);
 }
 
 // TODO: Certain large input sizes cause the QNN graph to fail to finalize with error 1002 (QNN_COMMON_ERROR_MEM_ALLOC).
-TEST_F(QnnHTPBackendTests, DISABLED_TestMaxPool_Large_Input2_Ceil_HTP_u8) {
-  RunQDQMaxPoolOpTest<uint8_t>({1, 128, 16, 113},  // shape
-                               {2, 2},             // kernel_shape
-                               {2, 2},             // strides
-                               {0, 0, 0, 0},       // pads
-                               {1, 1},             // dialations
-                               1,                  // ceil_mode
-                               0,                  // storage_order
-                               "NOTSET",           // auto_pad
+TEST_F(QnnHTPBackendTests, DISABLED_MaxPool_Large_Input2_Ceil_HTP_u8) {
+  RunQDQMaxPoolOpTest<uint8_t>(TestInputDef<float>({1, 128, 16, 113}, false, -10.0f, 10.0f),  // Dynamic input with range [-10, 10]
+                               {utils::MakeAttribute("kernel_shape", std::vector<int64_t>{2, 2}),
+                                utils::MakeAttribute("strides", std::vector<int64_t>{2, 2}),
+                                utils::MakeAttribute("pads", std::vector<int64_t>{0, 0, 0, 0}),
+                                utils::MakeAttribute("dilations", std::vector<int64_t>{1, 1}),
+                                utils::MakeAttribute("ceil_mode", static_cast<int64_t>(1)),
+                                utils::MakeAttribute("storage_order", static_cast<int64_t>(0)),
+                                utils::MakeAttribute("auto_pad", "NOTSET")},
                                ExpectedEPNodeAssignment::All);
 }
 
 // TODO: Certain large input sizes cause the QNN graph to fail to finalize with error 1002 (QNN_COMMON_ERROR_MEM_ALLOC).
-TEST_F(QnnHTPBackendTests, DISABLED_TestMaxPool_LargeInput_1Pads) {
-  RunQDQMaxPoolOpTest<uint8_t>({1, 64, 384, 576},  // shape
-                               {3, 3},             // kernel_shape
-                               {2, 2},             // strides
-                               {1, 1, 1, 1},       // pads
-                               {1, 1},             // dialations
-                               0,                  // ceil_mode
-                               0,                  // storage_order
-                               "NOTSET",           // auto_pad
+TEST_F(QnnHTPBackendTests, DISABLED_MaxPool_LargeInput_1Pads) {
+  RunQDQMaxPoolOpTest<uint8_t>(TestInputDef<float>({1, 64, 284, 576}, false, -10.0f, 10.0f),  // Dynamic input with range [-10, 10]
```

Comment:
>1, 64, 284, 576 [](start = 52, length = 15)

the shape was [1, 64, 384, 576] previously, it's [1, 64, 284, 576] now. any reason to change 384 to 284?

Reply:
I'll change it back. That wasn't intentional. Thanks.

Reply:
Changed it back.